app/Exceptions/Handler.php

```diff
@@ -153,12 +200,61 @@ protected function mapException(\Throwable $e): \Throwable
 	}
 
 	/**
-	 * Renders the given HttpException.
+	 * Prepare a response for the given exception.
+	 *
+	 * This method is called by the framework, _after_ the framework has
+	 * decided that the client expects a HTML response, but _before_ the
+	 * actual work horse {@link Handler::renderHttpException} is called.
+	 *
+	 * This method is 99% identical to the parent method expects for a tiny
```

Comment:
```suggestion
	 * This method is 99% identical to the parent method except for a tiny
```

Fixing my own typos :grinning: